tests/uefistored/conftest.py

```diff
@@ -31,19 +37,18 @@ def uefi_vm_and_snapshot(imported_vm):
     vm.param_set('platform', 'secureboot', False)
     snapshot = vm.snapshot()
 
-    yield (vm, snapshot)
+    yield vm, snapshot
 
     snapshot.destroy()
 
-@pytest.fixture(scope='module')
-def uefi_vm(uefi_vm_and_snapshot):
-    vm, _ = uefi_vm_and_snapshot
-    yield vm
-
 @pytest.fixture(scope='module')
 def linux_uefi_vm(uefi_vm):
     vm = uefi_vm
     if vm.is_windows:
         pytest.skip("linux_uefi_vm can only be used on Linux VMs")
 
     yield vm
```

Comment:
While we're at refactoring, this fixture is not necessary anymore. It's just a combination of `unix_vm` (which will be available in global conftest once Gaël's PR is merged) and `uefi_vm`, and we can use them instead.

Reply:
Should it be named `unix_...` as well?

Reply:
Probably